devops/charts/bg-monitoring/README

```diff
@@ -1,8 +1,9 @@
-oc create secret generic rabbitmq-definitions-secret \
-    --from-file=rabbitmq.definitions.json=
+To install or upgrade helm chart in specific namespace and use all env variables
+helm upgrade -i ${HELM_RELEASE} --namespace=${OPENSIFT_NAMESPACE} -f  env/${HELM_STAGE}/env.yaml -f env/${HELM_STAGE}/secrets.yaml
```

Comment:
Seems like travis will fail to deploy. I don't see changes to "helm upgrade" run by it.